app/controllers/reports/regions_controller.rb

```diff
@@ -101,10 +101,21 @@ def details
 
     @repository = Reports::Repository.new(regions, periods: @period_range, follow_ups_v2: current_admin.feature_enabled?(:follow_ups_v2))
     chart_repo = Reports::Repository.new(@region, periods: chart_range, follow_ups_v2: current_admin.feature_enabled?(:follow_ups_v2))
+    medications_dispensation_service = MedicationDispensationService.new(region: @region, period: @period)
+
+    district_regions = if @region.state_region?
+      [@region, @region.district_regions].flatten
+    end
+
+    if @region.state_region?
+      @district_repository = Reports::Repository.new(district_regions, periods: @period_range, follow_ups_v2: current_admin.feature_enabled?(:follow_ups_v2))
+    end
```

Comment:
Let's remove this if we didn't add this.

Reply:
Came from this PR `Show overdue call result` #3126